### PR TITLE
C365632 Browse | Verify that the "Type of heading" facet option will display the name of facet option when zero results are returned (spitfire)

### PR DESCRIPTION
**Pull request by arsenhovh**
[Test Rail](https://foliotest.testrail.io/index.php?/cases/view/365632)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5647/)
image

### CODE REVIEW

File: cypress/e2e/marc-authority/verify-headingType-facet-option.cy.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import MarcAuthorityBrowse from '../../support/fragments/marcAuthority/MarcAuthorityBrowse';
+
+describe('Importing MARC Authority files', () => {
+  const testData = {};
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+  const fileName = '100_MARC_authority_records.mrc';
```
Comment (manowarqqq) [inline cypress/e2e/marc-authority/verify-headingType-facet-option.cy.js:17]: where is this file?

File: cypress/support/fragments/marcAuthority/marcAuthorities.js [hunk 2]
```diff
@@ -860,6 +860,25 @@ export default {
     });
   },
 
+  verifyHeadingTypeAccordionAccordionAndClick: () => {
+    cy.expect(headingTypeAccordion.exists());
+    cy.do(headingTypeAccordion.clickHeader());
+  },
+
+  chooseHeadingType: (headingType) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/marcAuthority/marcAuthorities.js:868]: It the same chooseTypeOfHeading()

File: cypress/support/fragments/marcAuthority/marcAuthorities.js [hunk 3]
```diff
@@ -860,6 +860,25 @@ export default {
     });
   },
 
+  verifyHeadingTypeAccordionAccordionAndClick: () => {
```
Comment (manowarqqq) [inline cypress/support/fragments/marcAuthority/marcAuthorities.js:863]: typo in naming
Not a useful method. If you need to open accordion to verify verifySelectedTextOfHeadingType just add click to this method

**Review by manowarqqq — COMMENTED** [review]
Please, for further tests, use rebase and amend to have  one commit before merging

File: cypress/e2e/marc-authority/verify-headingType-facet-option.cy.js (hunk 1 above)
Comment (arsenhovh) [reply cypress/e2e/marc-authority/verify-headingType-facet-option.cy.js:17]: it`s already in master (from other ticket)

File: cypress/support/fragments/marcAuthority/marcAuthorities.js (hunk 2 above)
Comment (arsenhovh) [reply cypress/support/fragments/marcAuthority/marcAuthorities.js:868]: chooseTypeOfHeading() you can choose even more than one 
chooseHeadingType() you can choose only one 
buy if you think i need to use chooseTypeOfHeading() , i will change

**Review by manowarqqq — APPROVED** [review]

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Pull request merged**